src/tallow.c

```diff
@@ -76,30 +97,70 @@ static void setup(void)
 	if (nocreate)
 		return;
 
-	/* init ipset and iptables */
-	/* delete iptables ref to set before the ipset! */
-	ext_ignore("%s/iptables -t filter -D INPUT -m set --match-set tallow src -j DROP 2> /dev/null", ipt_path);
-	ext_ignore("%s/ipset destroy tallow 2> /dev/null", ipt_path);
-	if (ext("%s/ipset create tallow hash:ip family inet timeout %d", ipt_path, expires)) {
-		fprintf(stderr, "Unable to create ipv4 ipset.\n");
-		exit(EXIT_FAILURE);
-	}
-	if (ext("%s/iptables -t filter -I INPUT 1 -m set --match-set tallow src -j DROP", ipt_path)) {
-		fprintf(stderr, "Unable to create iptables rule.\n");
-		exit(EXIT_FAILURE);
+	/* firewalld */
+	char fwd_path[PATH_MAX];
```

Comment:
Let me know if there is a better way to do this.

I did `fwd_pat` locally and only used it for the access test because I didn't really want to refactor `ipt_path` usage in the code and conf, at least not in this PR.  (Because `ipt_path` isn't really a path to iptables/ipset to being with, it's a path to sbindir.